src/main/java/net/sf/jabref/gui/groups/TransferableEntrySelection.java

```diff
@@ -11,7 +11,7 @@
 
 import net.sf.jabref.model.entry.BibEntry;
 
-class TransferableEntrySelection implements Transferable {
+public class TransferableEntrySelection implements Transferable {
```

Comment:
OK for me to move forward here. I wish the nasty UI issues of https://github.com/JabRef/jabref/pull/1637#issuecomment-246793569 can be solved.